.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: c
 script:
-  - ./autogen.sh
-  - ./configure
-  - make distcheck
+- ./autogen.sh
+- ./configure
+- make distcheck
+os:
+- linux
+- osx
+compiler:
+- gcc
+- clang
```

Comment:
If you don't expect doesn't mean it doesn't exist.

Clang may give useful warnings in build log, different from gcc.

And under osx make install fails, no xz there.